posts/urllib3-pipenv-standalone-test.rst

```diff
@@ -0,0 +1,114 @@
+This post compares the pytests of urllib3 to its nuitka-built .whl counterpart
+The results were very similar:
+	Regular pytests: ====== 3 failed, 836 passed, 456 skipped, 113 warnings in 47.54 seconds =======
+	Nuitka wheel pytests: ====== 1 failed, 838 passed, 456 skipped, 113 warnings in 47.59 seconds =======
+
+
+urllib3 normal pytests:
```

Comment:
I think this is going to look very ugly without proper block quoting. Checkout the ReST primer for how that is done.